File: week-2/exercicios/dezenas.py
"""
Exercicio 3
Faça um programa em Python que recebe um número inteiro e imprime seu dígito das dezenas.
Exemplo: Entrada 78615 | Saida 1
"""
numero = int(input("Digite um número inteiro:"))
dezenas = (numero //10) % 10
print('O dígito das dezenas é',dezenas)

File: week-2/exercicios/dezenas.cpp
#include<iostream>

int main() {
  int input;
  std::cout << "Digite um numero inteiro: ";
  std::cin >> input;

  int dezenas = (input / 10) % 10;
  std::cout << "O digito das dezendas e " << dezenas <<std::endl;
  return 0;
}
